integrations/spark/docker-compose.yml

```diff
@@ -1,13 +0,0 @@
-version: "3.7"
```

Comment:
Can you please put an explanation in the PR description why this is being removed?